setup.py

```diff
@@ -16,20 +16,15 @@
 history = open('CHANGES.rst').read()
 
 tests_require = [
-    'check-manifest>=0.25',
-    'coverage>=4.0',
+    'pytest-invenio>=1.4.0',
     'cryptography>=2.1.4',
-    'isort>=4.2.2',
     'mock>=1.3.0',
-    'pydocstyle>=1.0.0',
-    'pytest-cov>=1.8.0',
-    'pytest-pep8>=1.0.6',
-    'pytest>=3.8.0,<5.0.0',
+    'docker-services-cli>=0.2.0',
```

Comment:
Minor: you don't need this one, it should be installed by default from ``pytest-invenio``.